src/pages/StatisticsChart/common/index.tsx

```diff
@@ -200,13 +202,13 @@ export function SmartChartPage<T>({
   const content = query.isLoading ? (
     <ChartLoading show isThumbnail={isThumbnail} />
   ) : (
-    <ReactChartCore option={option} isThumbnail={isThumbnail} {...chartProps} />
+    <ReactChartCore option={option} isThumbnail={isThumbnail} {...chartProps} style={style} />
   )
 
   return isThumbnail ? (
     content
   ) : (
-    <ChartPage title={title} description={description} data={toCSV(dataList)}>
+    <ChartPage title={title} description={description} data={toCSV ? toCSV(dataList) : undefined}>
```

Comment:
`toCSV ? toCSV(dataList) : undefined` can be simplified as `toCSV?.(dataList)`

Ref:
- Optional chaining: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#syntax

Reply:
d3b5d91b